src/base/validation.lua

```diff
@@ -174,6 +175,13 @@
 	end
 
 
+	function m.actionSupportsToolset(prj)
+		if not p.action.supportsToolset(prj.language, prj.toolset) then
+			p.warn("Unsupported toolset '%s' used for language '%s' for project '%s'", prj.toolset, prj.language, prj.name)
```

Comment:
Most other validations use `warn` and not `error`...
So I stay coherent with above checks. (but I think all should be `error`).